ui/src/test/resources/paper_check_response.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Based on my previous comment, this file can be deleted

Reply:
removed